database/README.md

```diff
@@ -6,7 +6,59 @@ This folder contains the scripts to create the database, database tables(as show
 __Figure 1__ - _Starwatch Database ERD_: Shows the entity relationship diagram for the database used in this project.
 
 ## Setup
+### To Run Locally:
```

Comment:
Extensive explanation on how to run a similar RDS database in a similar fashion to StarWatch.